rules/php73/src/Rector/Case_/ConvertContinueToBreakRector.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Php73\Rector\Case_;
+
+use PhpParser\Node;
+use PhpParser\Node\Scalar\LNumber;
+use PhpParser\Node\Stmt;
+use PhpParser\Node\Stmt\Break_;
+use PhpParser\Node\Stmt\Case_;
+use PhpParser\Node\Stmt\Continue_;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Rector\Php73\Tests\Rector\Case_\ConvertContinueToBreakRector\ConvertContinueToBreakRectorTest
+ */
+final class ConvertContinueToBreakRector extends AbstractRector
+{
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition(
+            'Replaces continue statement with break',
+            [
+                new CodeSample(
+                    <<<'GOOD_SWITCH'
+switch(true) {
+    case 1:
+        continue;
+}
+GOOD_SWITCH,
+                    <<<'BAD_SWITCH'
+switch(true) {
+    case 1:
+        break;
+}
+BAD_SWITCH,
+                )
+            ]
+        );
+    }
+
+    public function getNodeTypes(): array
+    {
+        return [
+            Case_::class
+        ];
+    }
+
+    /**
+     * @param Case_ $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        $statements = $node->stmts;
+
+        foreach ($statements as $i => $statement) {
```

Comment:
I've not tried, but it should can be simplied with:

**Option 1: Reference value var**

```php
foreach ($node->stmts as &$statement) {
    if ($this->shouldReplaceWithBreak($statement)) {
         $statement = new Break_();
    }
}

return $node;
```

**Option 2: Using `traverseNodesWithCallable()`**

```php
        $this->traverseNodesWithCallable($node->stmts, function (Node $node) : void {
            if (! $this->shouldReplaceWithBreak($node)) {
                return;
            }

            $node = new Break_();
        });
        
        return $node;
```

Reply:
Better the 2nd option, references can go wrong very quickly

Reply:
Option 1 works but I personally prefer to avoid references at all cost to prevent (unwanted) side effects. Not sure what hte "standard" is for rector but if generally speaking everybody is fine/wants to use references I'm fine with changing it. (Is there maybe a benefit of reference over value that I'm not aware of?)
Option 2 didn't work for me, tried tinkering a bit with it but couldn't get it to work.

Reply:
I just tried with `traverseNodesWithCallable()` and it seems not working for re-apply node with new node. I think it is ok as is 👍